k8s-configuration/stockanalyzer.yaml

```diff
@@ -0,0 +1,62 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: stockanalyzer
+  labels:
+    app: stockanalyzer
+spec:
+  replicas: 1
```

Comment:
let's have at least `2`?

Reply:
I cannot just do it. My service uses third-party API and I can only fetch 50 records/min

Reply:
it looks like you need a distributed lock for your task. I expect that after introducing mongo you'll fix this
https://github.com/lukas-krecan/ShedLock

Reply:
I will check 'ShedLock' and add it with mongo migration. Okay?